docs/src/main/tut/typeclasses.md

```diff
@@ -0,0 +1,73 @@
+The typeclass pattern is a technique borrowed from haskell. You can
+think of it as an "interface" in the java sense, but universally
+quantified over some type. In scala they are encoded as a trait that
+has one type variable, such as:
```

Comment:
Multi-parameter type classes are a language extension in Haskell, and I can't think of too many Scala projects that use them (only Shapeless and Banana come to mind at the moment), but I agree that it makes sense to change the language here to be less categorical and more a description of typical practice.